src/backend/marsha/core/utils/medialive_utils.py

```diff
@@ -283,6 +290,12 @@ def stop_live_channel(channel_id):
 
 def create_mediapackage_harvest_job(video):
     """Create a mediapackage harvest job."""
+    request = requests.get(
```

Comment:
Is there a way to be notified of the failure unstead of testing this as other failures may still happen and even this is not a Garanty that the harvest will work?

Reply:
Yes we can be notified, this is what I made in #1062. I think both are needed.

Reply:
:+1: